openshift/catalog-sync/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: test-catalog-sync
```

Comment:
What's the reasoning behind adding a whole new Makefile? Does it not make sense to just add this to `openshift/Makefile`? It would make it easier to run the commands through the `openshift/release` CI if they're all using the same Makefile.

Reply:
These tests aren't directly related to OLMv1 components or its release.

The idea behind placing them under the catalog-sync directory was to keep everything related to those tests self-contained. That way, if we ever need to move or remove them in the future, it’ll be easier to manage without impacting other parts of the project. It also helps with maintainability and clarity.

Reply:
On top of that, we might need to add more targets in the future.